README.md

```diff
@@ -77,14 +80,15 @@ name= "httpbin post - returns post data"
 method= "POST"
 url="http://{{.base_service_url}}/post"
 wants_code= 200
-headers=["Content-Type: application/json"]
```

Comment:
You'll see this change elsewhere too. But this toml does not correct unmarshal to `map[string]string` as expected.

A table like this does though
```toml
[litmus.test.headers]
Content-Type = "application/json"
```